ggshield/sca/client.py

```diff
@@ -80,12 +80,16 @@ def scan_diff(
         self,
         reference: bytes,
         current: bytes,
+        scan_parameters: SCAScanParameters,
     ) -> Union[Detail, SCAScanDiffOutput]:
         result: Union[Detail, SCAScanDiffOutput]
         try:
             response = self._client.post(
                 endpoint="sca/sca_scan_diff/",
                 files={"reference": reference, "current": current},
+                data={
+                    "scan_parameters": SCAScanParameters.SCHEMA.dumps(scan_parameters)
+                },
```

Comment:
We need to check rthat the backend accepts this new input format and take it into account. For that, I think we need to:
1. Modify it is the backend side first
2. Regenerate all the sca related cassettes, as the cassette do not have an up-to-date request input now

Reply:
Since the serializer for the sca scan parameters in gim's backend is a pydantic model which allow extra fields, this works fine. It's juste that the response won't filter the vulns using the minimum severity used here.
There are tickets to implement that SCA-870 to 873, I think it's ok for now if we accept the changes in this PR (and the cassettes will be handled in those tickets).